lang/reader.rkt

```diff
@@ -1,8 +1,10 @@
 #lang s-exp syntax/module-reader r7rs
 
 #:wrapper1 (lambda (t)
-             (parameterize ([read-case-sensitive #f]
```

Comment:
Thanks for this... I don't know what I was thinking when I blindly copied all the R5RS reader settings.